### PR TITLE
Update migrate example error handling

### PR DESCRIPTION
**Pull request by tylerwray**
Per #100, seems like it should be the default to check if `err != migrate.ErrNoChange`. 

Adding it to the examples to help people out

### CODE REVIEW

File: migrate_test.go [hunk 1]
```diff
@@ -64,13 +64,13 @@ func TestNew(t *testing.T) {
 
 func ExampleNew() {
 	// Read migrations from /home/mattes/migrations and connect to a local postgres database.
-	m, err := New("file:///home/mattes/migrations", "postgres://mattes:secret@localhost:5432/database?sslmode=disable")
+	m, err := migrate.New("file:///home/mattes/migrations", "postgres://mattes:secret@localhost:5432/database?sslmode=disable")
```
Comment (dhui) [inline migrate_test.go]: Tests are already in the `migrate` package, so no need to import it.

Comment (tylerwray) [reply migrate_test.go]: Thanks, updated. 👍🏼

File: migrate_test.go [hunk 2]
```diff
@@ -70,7 +70,7 @@ func ExampleNew() {
 	}
 
 	// Migrate all the way up ...
-	if err := m.Up(); err != nil {
+	if err := m.Up(); err != migrate.ErrNoChange {
```
Comment (dhui) [inline migrate_test.go]: Same issue, don't need to specify the `migrate` package.
Also, you should make sure the `err != nil` as well.

**Comment by dhui** [comment]
Superseded by #336

**Pull request closed without merging**